grouper/repositories/permission_grant.py

```diff
@@ -111,6 +143,24 @@ def permission_grants_for_user(self, username):
         )
         return [PermissionGrant(g.name, g.argument) for g in group_permission_grants]
 
+    def service_account_grants_for_permission(self, name):
```

Comment:
Same notes as `group_grants_for_permission`

Reply:
As mentioned above, this turns out to not be an issue once we prevent disabling permissions that are granted to a service account, since service accounts remove all their grants when disabled anyway.